DemoApp/app/src/main/res/layout/activity_main.xml

```diff
@@ -10,51 +10,65 @@
     android:paddingBottom="@dimen/activity_vertical_margin"
     android:gravity="center_horizontal|bottom">
 
-    <LinearLayout
+    <RelativeLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
         android:gravity="center_horizontal|bottom"
         android:orientation="horizontal">
 
         <Button
             android:id="@+id/btnShowDiscoveryCenter"
-            android:layout_width="300px"
-            android:layout_height="120px"
+            android:layout_width="170dp"
+            android:layout_height="wrap_content"
             android:layout_margin="@dimen/btn_margin"
             android:background="@drawable/button_discovery"
-            android:text="Discovery center" />
+            android:text="Discovery center"/>
 
         <Button
             android:id="@+id/btnShowAppFeature"
-            android:layout_width="300px"
-            android:layout_height="120px"
-            android:layout_margin="@dimen/btn_margin"
+            android:layout_width="170dp"
+            android:layout_height="wrap_content"
             android:background="@drawable/button_interstitial"
-            android:gravity="center"
-            android:text="App Feature" />
+            android:text="App Feature"
+            android:layout_margin="@dimen/btn_margin"
+            android:layout_toEndOf="@+id/btnShowDiscoveryCenter" />
 
         <Button
             android:id="@+id/btnShowDirectEngagement"
-            android:layout_width="300px"
-            android:layout_height="120px"
+            android:layout_width="170dp"
+            android:layout_height="wrap_content"
             android:layout_margin="@dimen/btn_margin"
             android:background="@drawable/button_engagement"
-            android:text="Direct Engagement" />
+            android:text="Direct Engagement"
+            android:layout_toEndOf="@id/btnShowAppFeature"/>
 
         <Button
             android:id="@+id/btnShowVideo"
-            android:layout_width="300px"
-            android:layout_height="120px"
+            android:layout_width="170dp"
+            android:layout_height="wrap_content"
             android:layout_margin="@dimen/btn_margin"
             android:background="@drawable/button_video"
-            android:text="Video" />
+            android:text="Video"
+            android:layout_toEndOf="@id/btnShowDirectEngagement"/>
 
         <Button
```

Comment:
Please remove the random ad button, as it serves no purpose (also from the code).